core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1602,6 +1602,15 @@ class SparkContext(config: SparkConf) extends Logging {
     }
   }
 
+  /**
+   * Get the max number of tasks that can be concurrent launched currently.
```

Comment:
How about like this?
```
 * Get the max number of tasks that can be concurrently launched when the method is called.
 * Note that please don't cache the value returned by this method, because the number can be
 * changed due to adding/removing executors. 
```